projects/arc-lib/README.md

```diff
@@ -1,35 +1,31 @@
-# Sourceloop Shared Library 
-
-<!-- DOCUMENTATION -->
+# arc Shared-Library 
```

Comment:
Titles are always in capital case, and use uppercase for sort forms. 
For eg. ARC instead of arc.
Use can you tools like: https://convertcase.net/
To get it quickly.

Reply:
done